gm4_smelteries/data/gm4_smelteries/loot_tables/smelt.json

```diff
@@ -0,0 +1,146 @@
+{
+    "type": "minecraft:fishing",
+    "pools": [
+        {
+            "rolls": 1,
+            "entries": [
+                {
+                    "type": "minecraft:alternatives",
+                    "children": [
+                        {
+                            "type": "minecraft:item",
+                            "name": "minecraft:gold_ingot",
+                            "conditions": [
+                                {
+                                    "condition": "minecraft:match_tool",
+                                    "predicate": {
+                                        "item": "minecraft:gold_ore"
+                                    }
+                                }
+                            ]
+                        },
+                        {
+                            "type": "minecraft:item",
+                            "name": "minecraft:iron_ingot",
+                            "conditions": [
+                                {
+                                    "condition": "minecraft:match_tool",
+                                    "predicate": {
+                                        "item": "minecraft:iron_ore"
+                                    }
+                                }
+                            ]
+                        },
+                        {
+                            "type": "minecraft:item",
+                            "name": "minecraft:coal",
+                            "conditions": [
+                                {
+                                    "condition": "minecraft:match_tool",
+                                    "predicate": {
+                                        "item": "minecraft:coal_ore"
+                                    }
+                                }
+                            ]
+                        },
+                        {
+                            "type": "minecraft:item",
+                            "name": "minecraft:lapis_lazuli",
+                            "conditions": [
+                                {
+                                    "condition": "minecraft:match_tool",
+                                    "predicate": {
+                                        "item": "minecraft:lapis_ore"
+                                    }
+                                }
+                            ]
+                        },
+                        {
+                            "type": "minecraft:item",
+                            "name": "minecraft:diamond",
+                            "conditions": [
+                                {
+                                    "condition": "minecraft:match_tool",
+                                    "predicate": {
+                                        "item": "minecraft:diamond_ore"
```

Comment:
So, I'm not so sure how I feel about the stone-type ores being doubled. In real-life, gemstones would be destroyed by the high-heat of a furnace. Redstone and Lapis already drop tons so this feels like kinda a loss? And coal is, well flammable. 
They seem almost like extraneous features that aren't needed? I don't know. Discussion on this would be good.

Reply:
I agree tbh. IRL getting metal from ore involves heating the ore in a furnace, causing the metal to melt and letting you separate the rock, metal and slag into different layers so that you can extract the pure metal. I imagine smelteries basically being a more efficient setup than a furnace which is able to extract metal from the ore with less lost material. Gemstones and minerals are cut from the rock and their purity is sort of fixed.

Both from a lore position and a gameplay one where non-metals can be fortuned, I don't think diamonds, emeralds or lapis should be doubleable. Redstone is a powder so it feels more like something a breaking machine would do.

Reply:
Sparks convinced me. I think only gold, iron, sand, and chorus should be doubleable.